pnpm-lock.yaml

```diff
@@ -7136,6 +7136,9 @@ packages:
   axios@1.8.2:
```

Comment:
`1.8.2` is still in the lockfile

Reply:
This is still in the lockfile. so, the scanners will continue to complain until we actually get rid of this.